inloop/solutions/views.py

```diff
@@ -29,54 +29,111 @@ def get(self, request, id):
         return JsonResponse({'solution_id': solution.id, 'status': solution.status()})
 
 
-class SolutionEditorView(LoginRequiredMixin, View):
+class SolutionSubmissionView(LoginRequiredMixin, View):
```

Comment:
I think the name is misleading, this class is not a view, but just a base class with some methods that are used in views. Probably should also end in `Mixin`.